File: FMAplication/Domain/Sales/SalesPointReceivedTransfer.cs
using FMAplication.Core;
using FMAplication.Enumerations;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FMAplication.Domain.Sales
{
    public class SalesPointReceivedTransfer:AuditableEntity<int>
    {
        public string TransactionNumber { get; set; }
        public string Remarks { get; set; }
        public bool IsConfirmed { get; set; }
        public TransactionStatus TransactionStatus { get; set; }
        public DateTime TransactionDate { get; set; }
        public int FromSalesPointId { get; set; }
        public int ToSalesPointId { get; set; }
        public int SourceTransferId { get; set; }
        public List<SalesPointReceivedTransferItem> Items;
    }
}

File: FMAplication/ClientApp/src/app/Shared/Entity/Sales/salesPointReceivedTransfer.ts
import { SalesPoint } from ".";
import { SalesPointTransfer } from "./salesPointTransfer";
import { SalesPointReceivedTransferItem } from "./salesPointReceivedTransferItem";

export class SalesPointReceivedTransfer{
    id: number;
    transactionNumber: string;
    remarks: string;
    isConfirmed: boolean;
    transactionStatus: number;
    transactionDate: string;
    transactionDateStr: string;
    fromSalesPointId: number;
    fromSalesPoint: SalesPoint;
    toSalesPointId: number;
    toSalesPoint: SalesPoint;
    sourceTransferId: number;
    items: SalesPointReceivedTransferItem[];
    sourceTransfer: SalesPointTransfer;
}